src/cljs/org/broadinstitute/firecloud_ui/page/method_repo.cljs

```diff
@@ -53,9 +47,11 @@
    :component-did-mount
    (fn [{:keys [state]}]
      (endpoints/call-ajax-orch
-       {:endpoint endpoints/list-methods
+       {:endpoint endpoints/list-configurations
         :on-done (fn [{:keys [success? get-parsed-response status-text]}]
                    (if success?
-                     (swap! state assoc :methods (get-parsed-response))
+                     (do
+                       (utils/rlog (str "methods are " (get-parsed-response)))
```

Comment:
Is this just for testing or do we intend to log these in production? If just for testing, then remove.